checkov/github/dal.py

```diff
@@ -49,7 +49,7 @@ def _headers(self):
     def get_branch_protection_rules(self):
         if self.current_branch and self.current_repository:
             branch_protection_rules = self._request(
-                endpoint="repos/{}/branches/{}/protection".format(self.current_repository, self.current_branch),
+                endpoint="repos/{}/{}/branches/{}/protection".format(self.repo_owner, self.current_repository, self.current_branch),
```

Comment:
```suggestion
                endpoint=f"repos/{self.repo_owner}/{self.current_repository}/branches/{self.current_branch}/protection"
```
I think leveraging f-string here, makes it a bit more readable 🙂